### PR TITLE
adding docs for loss functions, Module, Container and fix typos

### PR DESCRIPTION
**Pull request by soumith**
adding docs for 

- loss functions
  - some of them have examples and some dont, for now.
- nn.Module
- nn.Container

fix typos

### CODE REVIEW

**Review by apaszke — COMMENTED** [review]
Eh, I put all comments in the auto-generated files, sorry. Btw can we remove them from the repo pls?
